.github/workflows/linux-eic-shell.yml

```diff
@@ -2,6 +2,10 @@ name: linux-eic-shell
 
 on:
   push:
+    branches:
+      - main
+    tags:
+      - '*'
   pull_request:
```

Comment:
Every job except for `deploy-docs` (which only runs on `main`) still runs on PRs though.. is that what you want?

Reply:
Yeah, we still want to test pull requests. Mainly we don't want to trigger juggler builds every day.